packages/x-data-grid/src/components/virtualization/GridBottomContainer.tsx

```diff
@@ -20,10 +23,30 @@ const Element = styled('div')({
 export function GridBottomContainer(props: React.HTMLAttributes<HTMLDivElement>) {
   const classes = useUtilityClasses();
 
+  const apiRef = useGridApiContext();
+  const { viewportOuterSize, minimumSize, hasScrollX, scrollbarSize } = useGridSelector(
+    apiRef,
+    gridDimensionsSelector,
+  );
+  const scrollHeight = hasScrollX ? scrollbarSize : 0;
+  const offset = Math.max(
+    viewportOuterSize.height -
+      minimumSize.height -
+      // Subtract scroll height twice to account for GridVirtualScrollerFiller and horizontal scrollbar
+      2 * scrollHeight,
+    0,
+  );
+
+  let style = props.style;
+  if (offset !== 0) {
+    style = { ...style, transform: `translateY(${offset}px)` };
+  }
```

Comment:
We don't use the HTML div props from the top/bottom container, we could remove the props parameter and avoid this block of code.

Reply:
Done for both top and bottom containers 👍🏻